src/main/java/net/fabricmc/loom/api/LoomGradleExtensionAPI.java

```diff
@@ -67,6 +67,8 @@ default void setAccessWidener(File file) {
 
 	Property<Boolean> getShareRemapCaches();
 
+	Property<Boolean> getUseLegacyMixinAp();
```

Comment:
I would put this in the MixinApExtensionAPI, possibly rename `MixinApExtensionAPI` to `MixinExtensionAPI` makes more sense to me to put all mixin related options under that.